tests/debugtopus.spec.ts

```diff
@@ -34,76 +44,114 @@ describe("prepareTestRun", () => {
-    async (numberOfLevelsToRoot) => {
-      let callCount = -1;
-      jest.mocked(existsSync).mockImplementation(() => {
-        callCount++;
-        return callCount === numberOfLevelsToRoot;
-      });
-
-      const appDir = "someDir";
-      const dir = getPackageRootLevel(appDir);
-      const levelUps = Array(numberOfLevelsToRoot)
-        .fill("")
-        .map(() => "..");
-
-      const expectedDir = path.join(appDir, ...levelUps);
-      expect(dir).toEqual(expectedDir);
-    },
-  );
-
-  it.each([5, 6, 10, 100])(
-    "raises error when %s levels",
-    async (numberOfLevelsToRoot) => {
-      let callCount = -1;
-      jest.mocked(existsSync).mockImplementation(() => {
-        callCount++;
-        return callCount === numberOfLevelsToRoot;
-      });
+  describe(getPackageRootLevel.name, () => {
+    it.each([0, 1, 2, 3, 4])(
+      "gets root dir correctly",
+      async (numberOfLevelsToRoot) => {
+        let callCount = -1;
+        jest.mocked(existsSync).mockImplementation(() => {
+          callCount++;
+          return callCount === numberOfLevelsToRoot;
+        });
+
+        const appDir = "someDir";
+        const dir = getPackageRootLevel(appDir);
+        const levelUps = Array(numberOfLevelsToRoot)
+          .fill("")
+          .map(() => "..");
+
+        const expectedDir = path.join(appDir, ...levelUps);
+        expect(dir).toEqual(expectedDir);
+      },
+    );
 
-      const appDir = "someDir";
+    it.each([5, 6, 10, 100])(
+      "raises error when %s levels",
+      async (numberOfLevelsToRoot) => {
+        let callCount = -1;
+        jest.mocked(existsSync).mockImplementation(() => {
+          callCount++;
+          return callCount === numberOfLevelsToRoot;
+        });
+
+        const appDir = "someDir";
+
+        expect(() => getPackageRootLevel(appDir)).toThrowError(
+          "can't find root level node modules",
+        );
+      },
+    );
+  });
 
-      expect(() => getPackageRootLevel(appDir)).toThrowError(
-        "can't find root level node modules",
+  describe(getConfig.name, () => {
+    it("should replace broken octal escape sequences for windows paths", () => {
+      const brokenWindowsOutputPath = "some/broken/path/1234".replaceAll(
+        "/",
+        "\\",
       );
-    },
-  );
-
-  it("should replace broken octal escape sequences for windows paths", () => {
-    const brokenWindowsOutputPath = "some/broken/path/1234".replaceAll(
-      "/",
-      "\\",
-    );
 
-    const config = getConfig("doesn't/matter", brokenWindowsOutputPath);
+      const config = getConfig("doesn't/matter", brokenWindowsOutputPath);
 
-    expect(config).toContain('outputDir: "some\\\\broken\\\\path\\\\1234"');
+      expect(config).toContain('outputDir: "some\\\\broken\\\\path\\\\1234"');
+    });
   });
 
-  it("should not have \\ in command path", async () => {
-    const backslashPath = "C:\\User\\Some\\Path";
+  describe(createPlaywrightCommand.name, () => {
```

Comment:
tests here are the same aside from where I commented, they are just moved into "describes" for each method